native_client/BUILD

```diff
@@ -51,12 +51,28 @@ tf_cc_shared_object(
     copts = ["-Wno-sign-compare", "-fvisibility=hidden"],
     linkopts = select({
         "//tensorflow:darwin": [],
```

Comment:
Why don't we pass `-ldl -pthread` on macOS ?

Reply:
No idea, it works on my machine and passes tests.

Reply:
Hehe then why would it be required everywhere else? :D

Reply:
Maybe this https://stackoverflow.com/a/19382746